jdk12 openjdk hotspot test jobs fail

Applies to all jdk12 openjdk hotspot test jobs

${openjdkGit} seems not to be set.  In past we may have received this info from build pipelines, and now are relying on hard-coded values for git repo tags, 

https://github.com/AdoptOpenJDK/openjdk-tests/blob/master/openjdk_regression/build.xml#L26-L37 

While we can fix this temporarily by adding to build.xml, it would be good if this info could be passed to test jobs via build pipelines.

From [example console output](https://ci.adoptopenjdk.net/view/Test_all/job/openjdk12_hs_openjdktest_x86-64_linux/4/console): 
11:06:10  getOpenjdk:
11:06:10       [echo] git clone --depth 1 -q  https://github.com/AdoptOpenJDK/${openjdkGit}.git
11:06:10       [exec] fatal: unable to access 'https://github.com/AdoptOpenJDK/${openjdkGit}.git/': The requested URL returned error: 400
11:06:10       [exec] Result: 128
11:06:10  
11:06:10  BUILD FAILED
11:06:10  /home/jenkins/workspace/openjdk12_hs_openjdktest_x86-64_linux/openjdk-tests/TestConfig/scripts/build_test.xml:71: The following error occurred while executing this line:
11:06:10  /home/jenkins/workspace/openjdk12_hs_openjdktest_x86-64_linux/openjdk-tests/openjdk_regression/build.xml:110: The following error occurred while executing this line:
11:06:10  /home/jenkins/workspace/openjdk12_hs_openjdktest_x86-64_linux/openjdk-tests/openjdk_regression/build.xml:94: Warning: Could not find file /home/jenkins/workspace/openjdk12_hs_openjdktest_x86-64_linux/openjdk-tests/openjdk_regression/${openjdkGit} to copy.
11:06:10

--- a/openjdk_regression/build.xml
+++ b/openjdk_regression/build.xml
@@ -35,7 +35,10 @@
 	<condition property="openjdkGit" value="openjdk-jdk11u">
         <equals arg1="${JDK_VERSION}" arg2="11"/>
     </condition>
-	
+	<condition property="openjdkGit" value="openjdk-jdk12u">
+        <equals arg1="${JDK_VERSION}" arg2="12"/>
+    </condition>
+
 	<property environment="env" />
 	<if>
 		<isset property="env.RELEASE_TAG"/>

